app/layouts/default-layout.tsx

```diff
@@ -69,6 +69,18 @@ const DefaultLayout: React.FunctionComponent<Props> = ({
         </Container>
       </main>
       <Footer />
+      <style jsx>
+        {`
+          .page-wrap {
+            display: flex;
+            flex-direction: column;
+            min-height: 100vh;
+          }
+          main {
+            flex-grow: 1;
+          }
+        `}
+      </style>
```

Comment:
👌 Nice move - any particular reason for separating these styles out from the rest?

Reply:
They don't need to be global. There's probably more below that could be moved to this local style, but I left that for a future refactor.

Reply:
I thought so - just checking :)